src/pdf_beamtime_interfaces/srv/NewObstacleMsg.srv

```diff
@@ -0,0 +1,11 @@
+string name
+string type
+float64 x
+float64 y 
+float64 z 
+float64 w 
+float64 h 
+float64 d 
+float64 r 
```

Comment:
So all objects will have a w,h,d, and r? Or is this just over-specified to cover all bases?

Reply:
Only w, d and r are the over specified attributes. Kept for now as per the discussion over zoom on 10/19/2023.